libcaf_test/caf/test/runner.cpp

```diff
@@ -140,7 +148,11 @@ runner::parse_cli_result runner::parse_cli(int argc, char** argv) {
       format_to(err, "- {}\n", test_name);
     return {true, true};
   }
-  if (auto verbosity = get_as<std::string>(cfg, "verbosity")) {
+  if (auto suite_regex_string = get_as<std::string>(cfg_, "suites")) {
+    format_to(err, "suite regex: {}\n", *suite_regex_string);
+    return {true, true};
+  }
```

Comment:
I'm not sure if I understand what you are trying to do here. Isn't this failing the run if "suites" is defined? We also parse this option later again.